### ISSUE TITLE
Improve OCR

### ISSUE THREAD

**Issue by Snehal-Reddy** [Snehal-Reddy/MedAI#3]
The current OCR is depends on Azure Vision API and is not open sourced. Integrate open source OCR.

**Comment by jay24rajput** [comment]
Hello, can this issue be solved under KWOC?? If yes, I have an idea for the same

**Comment by Snehal-Reddy** [comment]
Yeah sure ! Join the slack channel (http://bit.ly/2LCeuEW) and we can continue the discussion there.
